scheduler/src/cook/mesos.clj

```diff
@@ -61,7 +63,8 @@
                      base-wait 500 ; millis
                      opts {:retry-schedule (cook.util/rand-exponential-seq retries base-wait)}]]
          (if (and (<= memory 200000) (<= ncpus 32))
-           (dutils/transact-with-retries!! conn opts (concat [txn] additional-txns-per-job))
+           (timers/time! submit-to-mesos-duration
```

Comment:
We should also have a counter of number of jobs submitted, and perhaps a timer that is normalized for the number of jobs run (submit time / job ). One would hope submit time / job == `submit-to-mesos-duration`. Even if the submit time is linear, we can start to reason about how long a submission of n jobs *should* take.